code-challenges.js

```diff
@@ -10,45 +10,154 @@
 
-var myString1 = "bravo"
-var myString2 = "charlie"
-
-
+// var myString1 = "bravo"
+// var myString2 = "charlie"
+
+// var newString3 = myString1.split("")
+// var newString4 = newString3.reverse()
+// var newString5 = myString2.split("")
+// var newString6 = newString5.reverse()
+// var newString7 = newString4.join("")
+// var newString8 = newString6.join("")
+// console.log(newString7, newString8)
+/////////////////////////////////////////////////////////////////////// Todd's Pseudocode //////////////////////////////////////////////////////////////
+// First thing I had to do was convert the strings into an array because the .reverse is an a ray method.
+// As you can see I worked with one and then the other just to make things simple for me.
+// Once I got them into a race I then reversed them and then using the .join Method I was then able to convert them back into a string.
+// Then I worked on the second bit of code for Charlie once I knew that I had bravo correct.
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // --------------------4) Create a function that takes in two numbers and subtracts the smaller number from the larger number.
 // Use the two sets of test variables provided below. Expected output: 15, 12
 
-var numberExample1 = 42
-var numberExample2 = 27
-
-var numberExample3 = 7
-var numberExample4 = 19
-
+// var numberExample1 = 42
+// var numberExample2 = 27
+
+// var numberExample3 = 19
+// var numberExample4 = 17
+
+// const equation1 = (answer) => {
+//     if(numberExample1 > numberExample2) {
+//         return numberExample1 - numberExample2
+//     } else if (numberExample1 < numberExample2) {
+//         return `${numberExample1} is smaller then ${numberExample2} so therefore I cannot compute`
+//     }
+// }
+// console.log(equation1())
+
+// const equation2 = (answer) => {
+//     if(numberExample3 > numberExample4) {
+//         return numberExample3 - numberExample4
+//     } else if (numberExample3 < numberExample4) {
+//         return `${numberExample3} is smaller then ${numberExample4} so therefore I cannot compute`
+//     }
+// }
+// console.log(equation2())
+/////////////////////////////////////////////////////////////////////// Todd's Pseudocode //////////////////////////////////////////////////////////////////
+// This is another question where I knew the answer and then I started typing and getting ahead of myself and not really visualizing how the code was going to look and in what order.
+// So they were a few stumbling blocks that I had to slow down and really think through it.
+// Then I saw that Sara One of the instructors were on and asked for help.
+// The only hint she gave me was to check my parentheses.
+// As I said before I was just in too big of a hurry and totally missed a set of parentheses in my consul log. 
+// Once I had fix that problem the code ran perfectly. 
```

Comment:
So there is a really cool history here that we could get into but suffice to say, mathematics has an idea where we can pass multiple parameters into a single function. In programming we can and often do,  the same thing. 

const equation = (input1, input2) => { return input1 - input2 } or something similar